packages/core/README.md

```diff
@@ -22,9 +22,15 @@ mechanisms, but a few are supplied. See Capturing Function Calls below.
 The AWS X-Ray SDK has two modes: `manual` and `automatic`.
 By default, the SDK is in automatic mode. You can flip the mode of the SDK using the following:
 
-    AWSXRay.enableManualMode();
+```js
+AWSXRay.enableAutomaticMode();
+
+AWSXRay.enableManualMode();
+
+process.env.AWS_XRAY_MANUAL_MODE = 'true';
+```
```

Comment:
Maybe something like this to keep the comment directly above the `AWS_XRAY_MANUAL_MODE` env variable?
```suggestion
```js
AWSXRay.enableAutomaticMode();

AWSXRay.enableManualMode();

// see https://github.com/aws/aws-xray-sdk-node/pull/595 for details on using this environment variable to prevent memory leaks when using manual mode
process.env.AWS_XRAY_MANUAL_MODE = 'true';
```
```